app/controllers/tickets_controller.rb

```diff
@@ -153,40 +152,37 @@ def details_before_anonymization
                                      .map { |checks| checks.map(&:first) }
 
     render json: {
-      user: user,
-      person: person&.as_json(private_attributes: person_private_attributes),
+      user: @user,
+      person: @person&.as_json(private_attributes: person_private_attributes),
       action_items: action_items,
       non_action_items: non_action_items,
       message_args: message_args,
     }
   end
 
   def anonymize
-    user, person = user_and_person_from_params
-    return if check_errors(user, person)
-
-    if user&.banned?
+    if @user&.banned?
```

Comment:
Would we want to turn this into a before_action as well? Or do we only want to use before_actions in the case that they will apply to multiple endpoints?

Reply:
There is room for improvement here, however some of these business-logic checks can only be executed once the controller has started to process some data already.

Right here, this line is a candidate for extracting. But it's not strictly necessary in the scope of this PR, whose purpose is to make the RuboCop linter happy for a version bump.